src/utils/globals.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+export default function setElementsVersion() {
```

Comment:
Since this is in a generic `globals` util, should this be a named export rather than the deafult?